firebase_admin/ml.py

```diff
@@ -633,6 +644,183 @@ def as_dict(self, for_upload=False):
         return {'automlModel': self._auto_ml_model}
 
 
+class TFLiteManagedUploadSource(TFLiteModelSource):
+    """TFLite model source representing a TFLite model uploaded in the console. (Output only)."""
+    def __init__(self):
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__)
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+    @property
+    def managed_upload(self):
+        return True
+
+    def as_dict(self, for_upload=False):
+        # This class is output only, you can't create
+        # a model from a Managed Upload Source since that
+        # indicates it was created through the console, and
+        # this is not the console.
+        # (the backend will ignore the managed_upload field)
+        return {'managedUpload': True}
+
+
+class CoreMlFormat(ModelFormat):
+    """Model format representing a Core ML model.
+
+    Args:
+      model_source: A CoreMlModelSource sub class. Specifies the details of the model source.
+    """
+    def __init__(self, model_source=None):
+        self._data = {}
+        self.model_source = model_source
+
+    @classmethod
+    def from_dict(cls, data):
+        """Create an instance of the object from a dict."""
+        data_copy = dict(data)
+        coreml_format = CoreMlFormat(model_source=cls._init_model_source(data_copy))
+        coreml_format._data = data_copy  #pylint: disable=protected-access
+        return coreml_format
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
```

Comment:
You can avoid the lint error by just comparing the public properties. That will also make the behavior less dependent on `_data`.

```
return isinstance(other, self.__class__) and
  self.model_source == other.model_source and
  self.size_bytes == other.size_bytes
```